src/Uno.UI/UI/Xaml/Controls/Popup/IPopup.cs

```diff
@@ -13,11 +13,13 @@
 
 namespace Windows.UI.Xaml.Controls
 {
-    public interface IPopup
-    {
+	internal interface IPopup
```

Comment:
Breaking

Reply:
There was already another discussion about this one. IPopup doesn't exists in UWP.

Reply:
Oups sorry I didn't saw it, and actually I prefer that we introduce breaking changes to match the UWP contract! However as you checked the "Contains **NO** breaking change", just to be aware there one ;)